commons/addons/cellar-vs-fsbuckets.md

```diff
@@ -64,26 +64,26 @@ data you put in it.
 
 Cellar is an Amazon S3 API implementation using [Ceph Rados Gateway](https://ceph.com/ceph-storage/object-storage/).
 It provides an HTTP API that can be used with any S3 API Client.
-Clients exists for a lot of languages.
+Clients exist for a lot of languages.
 You can use any S3 Client, from the official Amazon ones to the community ones.
 
-If you already use S3, you will be in your element.
+If you already using S3, you will be in your element.
 
 To store a file, issue a PUT HTTP request to Cellar's API.
 To get a file, issue a GET request to the API.
 You need secret and public keys to authenticate your requests.
-You can manage ACLs, generate signed URLs with an expiration date.
+You can manage ACLs, and generate signed URLs with an expiration date.
```

Comment:
Remove comma if you add "and".

Reply:
```suggestion
You can manage ACLs and you can generate signed URLs with an expiration date.
```
It is a bit repetitive but it is better to separate the two independents ideas, isn't it?